packages/react/src/ActionList/ActionList.module.css

```diff
@@ -213,6 +213,22 @@
       }
     }
 
+    &:where([data-last-selected]:not(:last-of-type)) {
```

Comment:
I like this CSS-only solution for visually separating selected and unselected items, but where did this design change come from? We didn't do this before.

Reply:
here's the issue https://github.com/github/primer/issues/2409!